dashboard/app/serializers/script_lessons_serializer.rb

```diff
@@ -0,0 +1,17 @@
+module ScriptLessons
+  class LessonSerializer < ActiveModel::Serializer
+    attributes :key, :properties
```

Comment:
nice find on the ActiveModel Serializer! This looks like a great approach overall.